packages/core/admin/admin/src/content-manager/components/Wysiwyg/Editor.js

```diff
@@ -25,9 +25,8 @@ const Editor = ({
       lineWrapping: true,
       extraKeys: {
         Enter: 'newlineAndIndentContinueMarkdownList',
-        // Leaving this commented for now
-        // Tab: false,
-        // 'Shift-Tab': false,
+        Tab: false,
```

Comment:
I am not so sure about this, I think it makes it harder to support nested list as the user has to know that he needs two indent.